src/redux/__tests__/Rockets.test.js

```diff
@@ -0,0 +1,16 @@
+import rocketReducer, { fetchRockets } from '../rockets/rockets';
+
+describe('Test reducer and actions', () => {
+  it('Returns initial state with no action', () => {
+    expect(rocketReducer(undefined, {})).toEqual([]);
+  });
+
+  it('Updates state correctly based on the given rocket', () => {
+    const rockets = [{
+      id: '1',
+      name: 'some name',
+    }];
+
+    expect(rocketReducer([], fetchRockets(rockets))).toEqual(rockets);
+  });
+});
```

Comment:
It would be good if you could test the action for booking a rocket. This test is only testing if we get update state with new rockets.

It would be good to add this test in this PR since it is related to this PR.

This is not blocking requirement for this PR to be merged but if not done here, it should be added either irrelevant PR or in a dedicated task for writing tests.